src/machine.rs

```diff
@@ -172,6 +172,38 @@ impl OlmMachine {
         future_to_promise(async move { Ok(me.display_name().await?) })
     }
 
+    /// Whether automatic transmission of room key requests is enabled.
+    ///
+    /// Room key requests allow the device to request room keys that it might
+    /// have missed in the original share using `m.room_key_request`
+    /// events.
+    #[wasm_bindgen(getter, js_name = "roomKeyRequestsEnabled")]
+    pub fn are_room_key_requests_enabled(&self) -> bool {
+        self.inner.are_room_key_requests_enabled()
+    }
+
+    /// Enable or disable automatic transmission of room key requests.
+    #[wasm_bindgen(setter, js_name = "roomKeyRequestsEnabled")]
+    pub fn toggle_room_key_requests(&self, enabled: bool) {
```

Comment:
(here and below for the other function) uber-nit: admittedly I'm being a bit pedantic here, but to me `toggle` carries the meaning of setting the value to `!value`. Would it make sense to replace the prefix with `set_` or `enable_` instead?